frontend/src/components/TopicCard/Links.js

```diff
@@ -25,7 +25,6 @@ class Links extends Component {
     this.props.updateTopic({
       metacode_id: metacodeId
     })
-    this.props.redrawCanvas()
```

Comment:
this comes out because there's already a backbone event that triggers a redraw when the topic model changes